tests/language/completion/pg_completion/test_naive_completion.py

```diff
@@ -80,38 +80,3 @@ def test_alter_well_known_keywords_completion(self):
             Completion(text="SYSTEM", display_meta='keyword'),
         ]))
         self.assertTrue(Completion(text="CREATE", display_meta="keyword") not in result)
-
-    def test_keyword_lower_casing(self):
```

Comment:
Why are we removing these tests?

Reply:
Casing is only checked if smart completion is on, so I moved these tests so the smart completion test file.